solr/docker/package/Dockerfile.local-package

```diff
@@ -0,0 +1,5 @@
+ARG BASE_IMAGE=alpine:3.12
```

Comment:
I'm confused why we need this as an intermediate docker image containing just the tgz? Also, why make this configurable and why not `FROM scratch`?

Reply:
Yeah, the idea was to allow for people to build in environments that don't have access to docker hub.

Didn't know about scratch, but it's perfect. Thanks for the suggestion.